django_project/base/models/project.py

```diff
@@ -105,7 +105,9 @@ class Project(models.Model):
     image_file = models.ImageField(
         help_text=_('A logo image for this project. '
                     'Most browsers support dragging the image directly on to '
-                    'the "Choose File" button above.'),
+                    'the "Choose File" button above. '
+                    'We recommend that you upload a logo of size 512 x 512 pixels '
+                    'and 2048 x 2018 pixels resolution.'),
```

Comment:
This text does not make any sense for me. Size is the physical dimension of the image. Resolution is how many DPI (Dots Per Inch) it is designed to be displayed at. Typically print uses 300dpi (though it can vary) and screen on older tech was 96dpi, newer devices much higher. As it is now this text will just confuse people. Why don't we just say "The ideal size for your image should be X x Y."

I'm also confused by your image ratio you have 2048x2018 - is that a typo? 20148 and 20148 maybe?

Reply:
Thanks @timlinux 
Am fixing that